### PR TITLE
Remove unused 'test' entries in the example pyr.json

### PR DESCRIPTION
**Pull request by ilkilic**
- (removed) Highlight feature labels in red when scores exceed the SCORE_THRESHOLD of 5.0.
- Throw an error if the checkpoints directory is empty.
- Remove unused 'test' entry from pyr.json.

### CODE REVIEW

**Comment by ilkilic** [comment]
After consulting with @darshanmandge , it was noted that highlighting the label has the potential to clutter the plot. Alternatively, we could consider implementing this as an optional setting, or keep only the scores in red as it was originally.

File: examples/emodel_pipeline_local_python/config/params/pyr.json [hunk 1]
```diff
@@ -49,16 +49,17 @@
...
-            {"name":"gSK_E2bar_SK_E2",        "val":[0, 0.1],       "test":0.0055436826094973947},
-            {"name":"decay_CaDynamics_DC0",   "val":[20, 300],       "test": 187.93939189992398},
-            {"name":"gamma_CaDynamics_DC0",   "val":[0.005, 0.05],  "test":0.018262044595632514}
+            {"name":"gNaTgbar_NaTg",          "val":[0, 1.5]},
+            {"name":"gNap_Et2bar_Nap_Et2",    "val":[0, 0.02]},
+            {"name":"gK_Pstbar_K_Pst",        "val":[0, 1]},
+            {"name":"gK_Tstbar_K_Tst",        "val":[0, 0.2]},
+            {"name":"gSKv3_1bar_SKv3_1",      "val":[0, 1]},
+            {"name":"gCa_HVAbar_Ca_HVA2",      "val":[0, 0.001]},
+            {"name":"gCa_HVAbar_Ca_HVA2",      "val":[0, 0.001]},
```
Comment (AurelienJaquier) [inline examples/emodel_pipeline_local_python/config/params/pyr.json:58]: you can remove this line, there are two of them

**Comment by AurelienJaquier** [comment]
How does changing the color have the potential to clutter the plot?

**Comment by ilkilic** [comment]
sorry, apparently we have a lot of optimizations with bad fittings scores. I believe that marking all the labels in red might be excessive.

**Review by AurelienJaquier — APPROVED** [review]
looks great!
when merging, give a name to the merge commit that is not misleading, since we ended up not highlighting labels

**Pull request merged**